backend/utils/seed.ts

```diff
@@ -0,0 +1,146 @@
+//seeds fake users and contacts into the database.
+import { faker } from '@faker-js/faker'
```

Comment:
Not something I'm worried about, but we'll probably want to not include this file (and faker js) in our production build (not that it really matters but good etiquette might suggest we do that). We can tackle that when we discuss production builds